Jenkinsfile

```diff
@@ -73,35 +114,31 @@ pipeline {
                                            secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
                                          ]]){
                             sh """
-                              make lint
-                              make test
-                              make integration-test LOGDNA_INGESTION_KEY=${LOGDNA_INGESTION_KEY}
+                              TEST_THREADS="${TEST_THREADS}" make integration-test LOGDNA_INGESTION_KEY=${LOGDNA_INGESTION_KEY}
                             """
                         }
                     }
-                    post {
-                        success {
-                            sh "make clean"
-                        }
-                    }
                 }
                 stage('Run K8s Integration Tests') {
```

Comment:
Is there an issue with running the K8s IT's as a separate thread?

Reply:
The side-by-side stages means they are run in parallel to the integration tests